flink/flink.sh

```diff
@@ -92,7 +92,8 @@ function install_flink_snapshot() {
   tar -xzvf "${flink_local}"
   rm "${flink_local}"
   # only the first match of the flink toplevel pattern is used
-  local files=( ${flink_toplevel_pattern} )
+  local files
+  mapfile -t files < <(compgen -G "${flink_toplevel_pattern}")
```

Comment:
This is too hard to understand.

Reply:
This is idiom recommended by https://github.com/koalaman/shellcheck